features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/AndroidTracer.kt

```diff
@@ -312,13 +313,13 @@ class AndroidTracer internal constructor(
 
         /**
          * Helper method to attach an error message to a specific Span.
-         * The error message will be added to the provided Span as a standard Error Tag.
+         * The error message will be logged with ERROR severity and can be seen in logs attached to the span.
```

Comment:
in facets it is called `status`

```suggestion
         * The error message will be logged with ERROR status and can be seen in logs attached to the span.
```

Reply:
now I use "status" everywhere